### PR TITLE
[NPUW] Use padding inside kokoro model 

### PR DESCRIPTION
**Pull request by sshumihi**
### Details:
**Problem**
When the Kokoro TTS model is reshaped to static input shapes in NPUW, the text_mask generation subgraph derives input_lengths from ShapeOf(input_ids), which becomes a constant equal to the static buffer size. This means the attention mask is always all-False ("attend everywhere"), so padding tokens after the actual phoneme sequence are never masked out. As a result, the BERT encoder, TextEncoder, and DurationEncoder process padding tokens as if they were real input, degrading audio quality and causing unnecessary computation in Model B.

**Solution**
1. Replace text_mask subgraph with an explicit Parameter 

2. Auto-fill text_mask from input_ids at inference time 
Scans the input_ids tensor to locate the EOS token (the Kokoro format is [BOS=0, tok₁, …, tokₙ, EOS=0, PAD, …]), computes the real sequence length, and fills the boolean mask: False for valid positions, True for padding — matching PyTorch semantics.

3. Zero out padding durations in pred_dur
After Model A inference, padding positions in the pred_dur output tensor are zeroed so they contribute zero frames to the audio, preventing Model B from wasting compute on padding tokens.

### Tickets:
 - E#203294

### CODE REVIEW

**Comment by sshumihi** [comment]
@OrestChura please have a look as well

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.hpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+/// Positions [real_len, full_len) are set to 0 so padded tokens contribute
+/// zero frames to the audio output.
+///
+/// @param pred_dur   Pointer to the pred_dur data.
+/// @param full_len   Total size of the pred_dur tensor.
+/// @param real_len   Number of valid tokens. Positions beyond this are zeroed.
+void zero_padding_durations(int64_t* pred_dur, std::size_t full_len, std::size_t real_len);
+
+/// @overload for int32 pred_dur tensors.
+void zero_padding_durations(int32_t* pred_dur, std::size_t full_len, std::size_t real_len);
```
Comment (OrestChura) [inline src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.hpp:49]: for 2 overloads, I think, it is all right, but if there will be more added in the future, with the differences only in the type - it should be changed to use template feature (with asserts on types or other ways of type control if needed)

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_split.cpp [hunk 2]
```diff
@@ -19,6 +19,51 @@
...
+    if (!text_mask_node) {
+        LOG_WARN("text_mask node not found — skipping attention mask replacement");
+        return;
+    }
+
+    LOG_DEBUG("replacing text_mask generation with Parameter input");
+
+    // text_mask shape is [1, seq_len] with element type BOOL
+    auto text_mask_param = std::make_shared<ov::op::v0::Parameter>(text_mask_node->get_output_element_type(0),
+                                                                   text_mask_node->get_output_partial_shape(0));
```
Comment (AsyaPronina) [inline src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_split.cpp:55]: The model should be already statically-shaped at this place or still has dynamic shapes?

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    return seq_len;  // no padding detected
+}
+
+void fill_text_mask_from_lengths(bool* mask_data, std::size_t seq_len, std::size_t real_len) {
+    for (std::size_t i = 0; i < seq_len; ++i) {
+        mask_data[i] = (i >= real_len);
+    }
+}
+
+void zero_padding_durations(int64_t* pred_dur, std::size_t full_len, std::size_t real_len) {
```
Comment (AsyaPronina) [inline src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.cpp:30]: Might be it can be templated?

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.hpp [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+/// Find the real (unpadded) sequence length in a Kokoro input_ids tensor.
+///
+/// The Kokoro input format is [BOS=0, tok1, ..., tokN, EOS=0, PAD, PAD, ...].
+/// This function scans from position 1 to find the first zero (EOS), and returns
+/// the real length including the EOS token itself.
+///
+/// @param input_ids Pointer to int64 input_ids data, shape [1, seq_len].
+/// @param seq_len   Total length of the input_ids sequence.
+/// @return          Number of valid tokens (including BOS and EOS).
```
Comment (AsyaPronina) [inline src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.hpp:22]: Is `BOS` really included?

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.cpp (hunk 3 above)
Comment (sshumihi) [reply src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.cpp:30]: I would like to avoid template to limit it to int64 / int32

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.hpp (hunk 4 above)
Comment (sshumihi) [reply src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_utils.hpp:22]: Yes, I printed it to check what we have, and indeed input sequence include BOS and EOS and input sequence = valid length.

File: src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_split.cpp (hunk 2 above)
Comment (sshumihi) [reply src/plugins/intel_npu/src/plugin/npuw/pipelines/kokoro/kokoro_split.cpp:55]: Should be already static shape, user has to convert it to static.
